extensions/amp-story/1.0/amp-story-page.js

```diff
@@ -1350,6 +1353,27 @@ export class AmpStoryPage extends AMP.BaseElement {
     );
   }
 
+  /**
+   * Checks if the page has any captions and toggles them.
+   * @private
+   */
+  togglePageCaptions_() {
+    this.hasLoadedVideoWith_((video) => video.querySelector('track')).then(
```

Comment:
`video.textTracks.length`, you don't need to query a `track`, this has a few other cool benefits

Reply:
`video` is an amp-video so it would need to be sth like `video.querySelector('video').textTracks` or `video.getImpl....video_.textTracks.length`, do you have a better way to get the video element from the amp-video so we can fetch the textTracks property? The property `textTracks` is very useful but only if we can access the HTMLVideoElement inside the amp-video easily.